lib/utils/flight_plan_utils.ts

```diff
@@ -105,23 +111,29 @@ export class FlightPlanUtils {
         if (fields[i].length > 2) {
           addRoute(decodeResult, fields[i].substring(2));       
```

Comment:
should this follow the other ones for consistency?

Reply:
Yes, and eventually we can extract out routing functions, etc.

Reply:
no - it goes right into the route without a colon